env.js

```diff
@@ -11,6 +11,7 @@ module.exports = function() {
     .option('--changes-limit [value]', 'number of changes to batch')
     .option('--postgres-table [value]', 'name of the postgres table to replicate to')
     .option('-d, --daemon', 'continually replicate between CouchDB and PostgresSQL')
+    .option('-s, --sleep', 'Sleep interval in milliseconds between runs in daemon mode (default 10 * 60 * 60 * 1000 [ms] = 10 hrs)')
```

Comment:
👍

Reply:
Can we come up with a better name for this parameter? I'm thinking `--sleep-interval` or `--polling-interval` or something more descriptive?

Secondly, `commander` allows setting defaults which would save a little code later on, eg:

```suggestion
    .option('-s, --sleep', 'Sleep interval in milliseconds between runs in daemon mode', 10 * 60 * 60 * 1000)
```

There are even more complicated options if you want to parse the variable or provide a human readable default value. More info in their [documentation](https://www.npmjs.com/package/commander).